grc/models.py

```diff
@@ -39,15 +37,7 @@ class Application(db.Model):
     filesCreated = db.Column(db.Boolean, default=False)
 
     def application_data(self) -> ApplicationData:
-        if self.user_input:
-            try:
-                application_data_obj: ApplicationData = jsonpickle.decode(self.user_input)
-            except:
-                application_data_obj = convert_weakly_typed_to_strongly_typed(ast.literal_eval(self.user_input))
-        else:
-            application_data_obj = ApplicationData()
-        
-        return application_data_obj
```

Comment:
I've run the "Check data migration" endpoint on all our environments.
This checks (for each row in the applications table) if we can successfully run `jsonpickle.decode(application.user_input)`.
There are no rows left (on my laptop, Sandbox, Staging or Production) where this line fails